astropy/io/ascii/cds.py

```diff
@@ -115,8 +127,12 @@ def get_cols(self, lines):
             match = re_col_def.match(line)
             if match:
                 col = core.Column(name=match.group('name'))
-                col.start = int(re.sub(r'[-\s]', '',
-                                       match.group('start') or match.group('end'))) - 1
+                col.start = (
+                    int(
+                        re.sub(r'[-\s]', '', match.group('start') or match.group('end'))
+                    )
+                    - 1
+                )
```

Comment:
```suggestion
                col.start = int(re.sub(r'[-\s]', '',
                                       match.group('start') or match.group('end'))) - 1  # fmt: skip
```
OK, starting with a try if this is the correct syntax to preserve the original. I note that
```
                col.start = int(
                    re.sub(r'[-\s]', '', match.group('start') or match.group('end'))) - 1  # fmt: skip
```
would also stay within the new 88-char limit; not sure which is preferable in terms of readability.